uber/models.py

```diff
@@ -1187,6 +1197,10 @@ def is_group_leader(self):
         return self.group and self.id == self.group.leader_id
 
     @property
+    def can_check_in(self):
```

Comment:
This isn't currently used, but will be used later to determine whether or not to display the 'check in' button.